inc/upgrades/namespace.php

```diff
@@ -352,6 +352,13 @@ function render_upgrade_widget() {
 	}
 
 	$latest = $releases[0];
+	$current = get_current_version_info();
+
+	// Return early for versions greater or equal to the latest version.
+	if ( $current['version'] >= $latest['version'] ) {
```

Comment:
I think we should fix the check inside `is_version_latest()` instead

Reply:
Thanks for the suggestion Ryan, definitely a better approach!